scene/realistic-lighting-and-shadows/src/main/resources/realistic_lighting_and_shadows/main.fxml

```diff
@@ -17,34 +17,25 @@
 <?import javafx.scene.layout.*?>
 
 <?import com.esri.arcgisruntime.mapping.view.SceneView?>
-<?import com.esri.arcgisruntime.mapping.view.LightingMode?>
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.control.Slider?>
 <?import javafx.scene.control.ComboBox?>
-<?import javafx.collections.FXCollections?>
 <StackPane fx:controller="com.esri.samples.realistic_lighting_and_shadows.RealisticLightingAndShadowsController"
            xmlns="http://javafx.com/javafx"
            xmlns:fx="http://javafx.com/fxml" stylesheets="/realistic_lighting_and_shadows/style.css">
     <!--SDK SceneView-->
     <SceneView fx:id="sceneView"/>
     <StackPane.margin>
-        <Insets topRightBottomLeft="10" />
+        <Insets topRightBottomLeft="10"/>
     </StackPane.margin>
     <!--Slider box-->
     <VBox styleClass="panel-region">
-        <Label fx:id="time"/>
+        <Label fx:id="timeLabel"/>
         <ComboBox fx:id="comboBox" maxWidth="Infinity">
-            <items>
-                <FXCollections fx:factory="observableArrayList">
-                    <LightingMode fx:value="NO_LIGHT"/>
-                    <LightingMode fx:value="LIGHT"/>
-                    <LightingMode fx:value="LIGHT_AND_SHADOWS"/>
-                </FXCollections>
-            </items>
         </ComboBox>
-        <Slider fx:id="timeSlider" value="12" max="24" majorTickUnit="4" minorTickCount="60" showTickMarks="true"
-                showTickLabels="true" onMouseDragged="#changeTimeOfDay">
+        <Slider fx:id="timeSlider" max="1440" majorTickUnit="240" minorTickCount="3600" value="720" showTickMarks="true"
+                showTickLabels="true" onMouseDragged="#updateTimeOfDay">
         </Slider>
```

Comment:
Nice work on updating the Slider. Remember you don't need an opening and closing tag if there's nothing between them. You could just write `<Slider />`. Same goes for the ComboBox above now that the values are added in the Controller class
```suggestion
        <Slider fx:id="timeSlider" max="1440" majorTickUnit="240" minorTickCount="3600" value="720" showTickMarks="true"
                showTickLabels="true" onMouseDragged="#updateTimeOfDay" />
```

Reply:
It also looks like you could move some of the styling of the slider into the CSS by adding the following to the `.slider` rule, and removing the respective params here:
```
  -fx-show-tick-labels: true;
  -fx-show-tick-marks: true;
  -fx-major-tick-unit: 240;
  -fx-major-tick-unit: 3600;
```
Better test it out though, since CSS can be a bit iffy with FXML. If it doesn't work easily happy to just leave this styling in the FXML